### PR TITLE
feat: delete verification records by orgId

### PR DESCRIPTION
**Pull request by bhavanakarwade**
### What?
- Implemented API to delete verification records by orgId
- Implemented store deleted audit trails functionality
- Error handling

### CODE REVIEW

File: apps/api-gateway/src/verification/verification.controller.ts [hunk 1]
```diff
@@ -33,6 +33,8 @@ import { User } from '../authz/decorators/user.decorator';
 import { GetAllProofRequestsDto } from './dto/get-all-proof-requests.dto';
 import { IProofRequestSearchCriteria } from './interfaces/verification.interface';
 import { ProofRequestType, SortFields } from './enum/verification.enum';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (tipusinghaw) [inline apps/api-gateway/src/verification/verification.controller.ts:36]: Any specific reason to disable `no-unused-vars` ?

File: apps/api-gateway/src/verification/verification.service.ts [hunk 2]
```diff
@@ -128,8 +129,8 @@ export class VerificationService extends BaseService {
         }
       }
 
-      async deleteVerificationRecord(orgId: string, userId: string): Promise<IVerificationRecords> {
-        const payload = { orgId, userId };
-        return this.sendNatsMessage(this.verificationServiceProxy, 'delete-verification-record', payload);
+      async deleteVerificationRecords(orgId: string, user: user): Promise<IVerificationRecords> {
```
Comment (tipusinghaw) [inline apps/api-gateway/src/verification/verification.service.ts:132]: Type and value should not same

**Review by KulkarniShashank — APPROVED** [review]
LG

**Review by tipusinghaw — APPROVED** [review]

File: apps/api-gateway/src/verification/verification.service.ts (hunk 2 above)
Comment (bhavanakarwade) [reply apps/api-gateway/src/verification/verification.service.ts:132]: Resolved

File: apps/api-gateway/src/verification/verification.controller.ts (hunk 1 above)
Comment (bhavanakarwade) [reply apps/api-gateway/src/verification/verification.controller.ts:36]: Resolved

**Pull request merged**